docs/examples/grafana/README.md

```diff
@@ -0,0 +1,211 @@
+# Grafana - All-in-one
+
+In this example, we are going to use [Prometheus](https://prometheus.io/) and [Tempo](https://grafana.com/oss/tempo/), both displayed inside [Grafana](https://grafana.com/grafana/).
+Just like [Jaeger example](../jaeger-docker/README.md), we will need to setup a collector to gather both metrics and traces.
+
+### Project setup
+
+Configure the project using your favorite tool, once again setup is similar to the Jaeger example, only difference being that we export metrics as they will be stored inside Prometheus:
+
+@:select(build-tool)
+
+@:choice(sbt)
+
+Add settings to the `build.sbt`:
+```scala
+libraryDependencies ++= Seq(
```

Comment:
This example uses Scala 3. Should we reference the Scala version in the settings too?

Reply:
I also thinking about whether we should provide multiple examples:
1) sbt. Scala 2
2) sbt. Scala 3
3) scala-cli. Scala 2
4) scala-cli. Scala 3

The code will be nearly identical, so the benefit is questionable.

WDYT?

Reply:
Regarding showing the differences between Scala 2 and 3 the only thing coming to my mind for sbt is `ThisBuild / scalaVersion`, maybe I'm missing something though. Assuming there's only this attribute changing I don't know if it's worth adding a dedicated scala 3 example just for that.
I never got to try scala-cli yet, so I can't really tell how different things should be there, so maybe it could be worth having a scala 3 example here, tell me what you think